.pre-commit-config.yaml

```diff
@@ -7,7 +7,7 @@
 # Common tasks
 #
 # - Run on all files:   pre-commit run --all-files
-# - Register git hooks: pre-commit install --install-hooks
```

Comment:
By default, `pre-commit` only installs `pre-commit` hook stage, but we also need `pre-push`